Gulpfile.js

```diff
@@ -33,6 +33,7 @@ const IS_CI = process.env.CI != null;
 const IS_WINDOWS = process.platform === "win32";
 
 export const ci = parallel(check, build, lint);
+export const test = series(clean, parallel(build, testBuild), testRun);
```

Comment:
I'd keep build sequence a series as well, to avoid any race conditions of writing the same files